detection_rules/etc/stack-schema-map.yaml

```diff
@@ -51,3 +51,7 @@
 "8.5.0":
   beats: "main"
   ecs: "8.3.1"
+
+"8.6.0":
+  beats: "main"
+  ecs: "8.4.0"
```

Comment:
Should this be `8.5.0`?

Reply:
8.3 as well

Reply:
We can update it later, but at the moment, `8.4.0` is the latest, while `8.5.0` is a release candidate as of today.